_posts/2017-12-18-34C3.md

```diff
@@ -7,10 +7,12 @@ categories: [talk, pinned]
 author: TheAssassin
 ---
 
+<img src="https://engelsystem.de/img/34c3_logo.png" class="postImg">
```

Comment:
Including external resources like this is a big privacy issue. I'd always avoid such stuff, and upload the image on the local server. Also, it doesn't matter from _where_ the file is served, there's no licensing or crediting information, hence the file potentially violates copyright law.

Also, there should be an empty line before the first paragraph, I'd say.